easybuild/easyconfigs/b/BerkeleyGW/BerkeleyGW-1.0.6-intel-2016.02-GCC-4.9.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'MakeCp'
+
+name = 'BerkeleyGW'
+version = "1.0.6"
+
+homepage = 'http://www.berkeleygw.org'
+description = """The BerkeleyGW Package is a set of computer codes that calculates the quasiparticle
+                 properties and the optical responses of a large variety of materials from bulk
+                 periodic crystals to nanostructures such as slabs, wires and molecules."""
+
+toolchain = {'name': 'intel', 'version': '2016.02-GCC-4.9'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['http://www.berkeleygw.org/releases/']
+sources = ['BGW-%(version)s.tar.gz']
+checksums = ['8740562da41e87a62eb8d457675e4bfd']
+
+patches = ['BerkeleyGW_ignore_archmk.patch']
+
+files_to_copy = ['*']
```

Comment:
@migueldiascosta using `ConfigureMake` together with `skipsteps = ['configure']` makes more sense, assuming you can somehow specify what the installation prefix should be...